realm/realm-library/src/androidTest/kotlin/io/realm/EmbeddedObjectsTest.kt

```diff
@@ -332,15 +412,27 @@ class EmbeddedObjectsTest {
     @Ignore("FIXME")
     fun copyToRealmOrUpdate_deleteReplacedObjects() {
         TODO()
-
     }
 
     @Test
-    @Ignore("Add in another PR")
     fun insert_noParentThrows() {
-        TODO()
+        realm.executeTransaction { realm ->
+            val child = EmbeddedSimpleChild("child")
+            assertFailsWith<IllegalArgumentException> { realm.insert(child) }
+        }
     }
 
+    @Test
+    fun dynamicRealm_createEmbeddedObjectWithUnmanagedParentThrows() =
```

Comment:
This doesn't throw because the parent is unmanaged, but because the parent is not a DynamicRealmObject. It isn't possible to have unmanaged DynamicRealmObjects currently, so not sure if it makes sense to keep this test.